qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
This files and processes are important because one acceptance test you should run is update from latest available release, so you need to bootstrap this with the latest rpm, deb, ... available to download.